modules/ztp-image-based-upgrade-prep-oadp.adoc

```diff
@@ -0,0 +1,136 @@
+// Module included in the following assemblies:
+// * edge_computing/image-based-upgrade/cnf-preparing-for-image-based-upgrade.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-image-based-upgrade-creating-backup-resources-with-ztp_{context}"]
+= Creating OADP resources for the image-based upgrade with GitOps ZTP
+
+You need to prepare your OADP resources to restore your application after an upgrade.
```

Comment:
Again, seems a bit closer to a statement of fact than a description of a process. 

Suggestion: "After an upgrade, prepare your OADP resources to restore your application."